modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * networking/switching-bf2-nic-dpu.adoc
+
+:_content-type: PROCEDURE
+[id="proc-switching-bf2-nic_{context}"]
+= Switching Bluefield-2 from DPU mode to NIC mode
+
+Use the following procedure to switch Bluefield-2 from data processing units (DPU) mode to network interface controller (NIC) mode.  
+
+.Prerequisites
+
+* 
```

Comment:
Need to add that sriov network operator needs to be installed first. The procedure to do that is already documented. A simple reference to that would be useful here.